### PR TITLE
use ByteString.empty

### PR DESCRIPTION
**Pull request by patriknw**
- and a few other cleanups

### CODE REVIEW

**Comment by drewhk** [comment]
LGTM, Great!

File: akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/ClusterMetricsRoutingSpec.scala [hunk 1]
```diff
@@ -61,7 +61,7 @@ object AdaptiveLoadBalancingRouterConfig extends MultiNodeConfig {
   // Extract individual sigar library for every node.
   nodeList foreach { role ⇒
     nodeConfig(role) {
-      ConfigFactory.parseString("akka.cluster.metrics.native-library-extract-folder=${user.dir}/target/native/" + role.name)
+      ConfigFactory.parseString(s"akka.cluster.metrics.native-library-extract-folder=$${user.dir}/target/native/" + role.name)
```
Comment (viktorklang) [inline akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/ClusterMetricsRoutingSpec.scala]: Wow, I didn't know $$ unescaped the dollar sign!

**Comment by ktoso** [comment]
Nice cleanup :) LGTM

**Comment by viktorklang** [comment]
:+1:

**Comment by patriknw** [comment]
PLS BUILD

**Pull request merged**